Assets/Mirage/Runtime/INetworkIdentityGenerator.cs

```diff
@@ -0,0 +1,10 @@
+namespace Mirage
+{
+    public interface INetworkIdentityGenerator
```

Comment:
can we can this `INetIdGenerator` instead since the field is called `NetId`

Reply:
done